### PR TITLE
Feature/force db consistency

### PR DESCRIPTION
**Pull request by denis-yuen**
Make and enforce consistency between JPA generated database and our migration scripts
* ways found to express certain schema we have in JPA with some effort
* uses an import.sql for last ditch schema not supported by JPA (needs explanation)
* travis-ci check for consistency

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,14 @@ jobs:
       - TESTING_PROFILE=unit-tests
       script: mvn --batch-mode clean install -P$TESTING_PROFILE
 
+    - stage: automated-review
+      jdk: openjdk10
```
Comment (coverbeck) [inline .travis.yml]: Should we be running locally with Open JDK 10? I'm still using Java 8 in my local dev environment.

File: dockstore-webservice/src/main/resources/migrations.1.6.0.xml [hunk 2]
```diff
@@ -235,4 +235,12 @@
                       tableName="event"/>
         <sql dbms="postgres">ALTER SEQUENCE organisation_id_seq RENAME TO organization_id_seq</sql>
     </changeSet>
+
+    <changeSet author="dyuen" id="maintain_consistency">
+        <!-- looks like the above attempt didn't take -->
```
Comment (coverbeck) [inline dockstore-webservice/src/main/resources/migrations.1.6.0.xml]: Yeah, I don't understand that... I had the same problem for the absolute path where it didn't seem to take with `<sql dbms` either. It's not clear to me why they have that.

File: .travis.yml (hunk 1 above)
Comment (denis-yuen) [reply .travis.yml]: I've been slipping in openjdk 10 stuff here and there ahead of a formal switch. 
I do want to switch long term from oracle 8 to openjdk 10/11

File: dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java [hunk 3]
```diff
@@ -61,7 +61,7 @@
         "This describes one entry in the dockstore. Logically, this currently means one tuple of registry (either quay or docker hub), organization, image name, and toolname which can be\n"
                 + " * associated with CWL and Dockerfile documents")
 @Entity
-@Table(uniqueConstraints = @UniqueConstraint(columnNames = { "registry", "namespace", "name", "toolname" }))
+@Table(uniqueConstraints = @UniqueConstraint(name = "ukbq5vy17y4ocaist3d3r3imcus", columnNames = { "registry", "namespace", "name", "toolname" }))
```
Comment (garyluu) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java]: constraint name

Comment (denis-yuen) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java]: I don't understand it, but I encountered issues dropping the constraint and re-adding it.
I think I'll limit this PR to matching existing structure and add improvements in a subsequent PR

**Pull request merged**